scripts/i18n-push.sh

```diff
@@ -21,9 +21,12 @@
 
 cd `dirname $BASH_SOURCE` && cd ..
 
+# Note we only extract the English language strings so they can be
+# sent to Transifex. All other language strings will be pulled
+# down from Transifex and so don't need to be generated.
 echo "Extracting i18n strings..."
-python manage.py makemessages --all
-python manage.py makemessages --all -d djangojs
+python manage.py makemessages -l en --all
+python manage.py makemessages -l en --all -d djangojs
```

Comment:
Why are you only extracting and pushing JS strings? There aren't strings in templates or Python files? https://docs.djangoproject.com/en/1.4/ref/django-admin/#django-admin-option---domain